src/ripple/app/misc/Validations.cpp

```diff
@@ -458,20 +467,27 @@ class ValidationsImp : public Validations
 
         mWriting = true;
         app_.getJobQueue ().addJob (
-            jtWRITE, "Validations::doWrite",
-            [this] (Job&) { doWrite(); });
+            jtWRITE, "Validations::queuedWrite",
+            [this] (Job&) { queuedWrite(); });
     }
 
-    void doWrite ()
+    void queuedWrite ()
     {
         auto event = app_.getJobQueue ().getLoadEventAP (jtDISK, "ValidationWrite");
```

Comment:
I think it saves load/timing information for the `ValidationWrite` job. `getLoadEventAP` returns a pointer to https://github.com/ripple/rippled/blob/develop/src/ripple/core/LoadEvent.h.  I think you need to hold onto the `event` for the duration of this block to capture that timing info.  Definitely a non-intuitive design.